cumulusci/tasks/salesforce/tests/test_UpdateAdminProfile.py

```diff
@@ -83,67 +85,68 @@
 </Profile>"""
 
 
-class TestUpdateAdminProfile(unittest.TestCase):
-    def test_run_task(self):
-        task = create_task(
-            UpdateAdminProfile,
-            {
-                "record_types": [
-                    {
-                        "record_type": "Account.HH_Account",
-                        "default": True,
-                        "person_account_default": True,
-                    }
-                ],
-                "namespaced_org": True,
-            },
-        )
-
-        def _retrieve_unpackaged():
-            profiles_path = os.path.join(task.tempdir, "profiles")
-            admin_profile_path = os.path.join(profiles_path, "Admin.profile")
-            os.mkdir(profiles_path)
-            with open(admin_profile_path, "w") as f:
-                f.write(ADMIN_PROFILE_BEFORE)
-
-        def _check_result():
-            with open(
-                os.path.join(task.tempdir, "profiles", "Admin.profile"), "r"
-            ) as f:
-                result = f.read()
-            self.assertMultiLineEqual(ADMIN_PROFILE_EXPECTED, result)
-
-        task._retrieve_unpackaged = _retrieve_unpackaged
-        task._deploy_metadata = _check_result
+def test_run_task():
```

Comment:
@jstvz is there any particular benefit to getting rid of the test class? I'm all for using pytest assertions instead of unittest, but I still find it kind of useful organizationally to keep the tests for methods of the same class grouped within one test class. Probably mostly a matter of personal preference though.

Reply:
I take your point, but I'm of the opinion the class isn't providing any additional organization since there's a 1:1 correspondence between the class under test and the module. 

In this case, the main reason for getting rid of `unittest.TestCase` is that it allows using pytest fixtures (`tmpdir`) and parameterization.